guidelines/enhancement_template.md

```diff
@@ -72,17 +72,33 @@ this proposal. Describe why the change is important and the benefits to users.
 
 ### User Stories
 
-Detail the things that people will be able to do if this is
-implemented. In each story, explain who the actor is based on their
-role, explain what they want to do with the system, and explain the
-underlying goal they have. Use the standard three part formula:
+Detail the things that people will be able to do if this is implemented and
+what goal that allows them to achieve .In each story, explain who the actor
+is based on their role, explain what they want to do with the system,
+and explain the underlying goal they have, what it is they are going to
+achieve with this new feature.
 
-"As a <role>, I want to <take some action> so that I can <accomplish a
-goal>."
+Use the standard three part formula:
+
+"As a \<role\>, I want to \<take some action\> so that I can \<accomplish a
+goal\>."
```

Comment:
rather than escaping these chars (which makes it messier to read when editing the text directly, as most EP authors are going to do), can we just make the whole block a code block, to avoid the rendering issue?

Reply:
Or block-quote with italics?

> Use the standard three part formula, replacing the sections in *italics* to explain the story:
>
> > As a *role*, I want to *take some action* so that I can *accomplish this goal*.

Or some such.

Reply:
I agree the escapes make it hard to read the template source while you're filling it in. I like Trevor's idea of using italics.